### PR TITLE
MLP-461/Bill Insurance

### PR DESCRIPTION
**Pull request by aginz**
Add documentation to explain how to add insurance to a patient, and use insurance to bill the appointment.

### CODE REVIEW

File: source/includes/entities/insurances/_create.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```json
+{"insurance"=>
+  { "id"=>2,
+    "user_id"=>2,
+    "member_id"=>"MDL684088116",
+    "insurance_payer_id"=>27,
+    "is_dependent"=>true,
+    "eligible"=>1
+  }
+}
```
Comment (sstaedt) [inline source/includes/entities/insurances/_create.md]: this is in Ruby form. we should modify it to be json.

File: source/includes/entities/insurances/_create.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+> When the patient is ineligible for telemedicine visits, the above command returns JSON structured like this:
+
+```json
+{"errors" => ['Member is not eligible']}
```
Comment (sstaedt) [inline source/includes/entities/insurances/_create.md]: same here

File: source/includes/entities/insurances/_create.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+To add an insurance payer to a patient, make a request to:
```
Comment (sstaedt) [inline source/includes/entities/insurances/_create.md]: This will also update. So we should say 'To add or update an insurance payer...'

**Comment by ahadida** [comment]
👍

**Pull request merged**